package.json

```diff
@@ -27,21 +27,21 @@
   },
   "dependencies": {
     "fs-mkdirp-stream": "^1.0.0",
-    "glob-stream": "^6.1.0",
+    "glob-stream": "^7.0.0",
     "graceful-fs": "^4.0.0",
-    "iconv-lite": "^0.4.24",
+    "iconv-lite": "^0.6.3",
     "is-valid-glob": "^1.0.0",
     "lazystream": "^1.0.0",
-    "lead": "^1.0.0",
+    "lead": "^3.0.0",
     "object.assign": "^4.0.4",
-    "pumpify": "^1.3.5",
-    "readable-stream": "^2.3.3",
+    "pumpify": "^2.0.1",
+    "readable-stream": "^3.6.0",
     "remove-bom-buffer": "^3.0.0",
     "remove-bom-stream": "^1.2.0",
```

Comment:
So, everything upgraded smoothing expect I have run into some issues with [remove-bom-stream v2](https://github.com/gulpjs/remove-bom-stream) which was published not too long ago.

I haven't delved too deep into it yet, but wanted to reach out if there's some early guidance for tackling this @phated? Is kinda the goal to move from `through2` to `streamx` across-the-board (per https://github.com/gulpjs/remove-bom-stream/issues/4)?

Reply:
Yes, almost the entirety of the gulp 5 holdup is finding someone to do the streamx updates across all projects.

Reply:
I see - would you be open to an intermediate release for some of these upgrades to `vinyl-fs` for now? Maybe leave the streamx work out for now but catch up on some of these other lagging dependencies. I left some other comment questions, but so far it doesn't seem like anything would constitute a major breaking change.

Reply:
Every major causes a major. All of the ones in the gulpjs org dropped node support. Everything leads to gulp 5 with no options for "intermediate"